docs/DeveloperGuide.md

```diff
@@ -178,6 +189,9 @@ The `Storage` component,
 
 Classes used by multiple components are in the `seedu.academydirectory.commons` package.
 
+### More Design Details of Version
```

Comment:
I assume this is placeholder for the future? If so, might want to add a TODO in this line so that we don't accidentally forget about it.